test/unit/UnitTestEconomicStrategy.ts

```diff
@@ -1,119 +1,246 @@
 import BigNumber from 'bignumber.js';
 import { assert } from 'chai';
-import { Config } from '../../src/index';
-import { mockConfig, mockTxRequest } from '../helpers';
-import { shouldClaimTx, getExecutionGasPrice, shouldExecuteTx } from '../../src/EconomicStrategy';
+import * as TypeMoq from 'typemoq';
+
+import { W3Util } from '../../src';
+import Cache, { ICachedTxDetails } from '../../src/Cache';
+import { IEconomicStrategy } from '../../src/EconomicStrategy';
+import {
+  EconomicStrategyManager,
+  IEconomicStrategyManager
+} from '../../src/EconomicStrategy/EconomicStrategyManager';
 import { EconomicStrategyStatus } from '../../src/Enum';
+import { ITxRequest } from '../../src/Types';
 
 describe('Economic Strategy Tests', () => {
-  let config: Config;
-  let txTimestamp: any;
+  let economicStrategyManager: IEconomicStrategyManager;
+
+  const account = '0x123456';
+  const defaultBalance = new BigNumber(10000000);
+  const defaultBounty = new BigNumber(10000000);
+  const defaultGasPrice = new BigNumber(10000000);
```

Comment:
Might be better to make this a single variable `const tenMwei = new BigNumber(10000000);`.